src/components/Components/CandidateSection/CandidateSection.scss

```diff
@@ -0,0 +1,40 @@
+.CandidateSection {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 96px 126px;
+    gap: 10px;
+    height: 992.38px;
+    background: var(--color-white-lilac);
+
+    &__wrapper__title {
+        height: 43px;
+        font-style: normal;
+        line-height: 133%;
+        color: var(--color-bunting);
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+
+        &__rec {
+            width: 24px;
+            height: 8px;
+            background: #0066FF;
```

Comment:
I think you can use custom css variable for the background color here, it is  --color-blue-ribbon